_posts/2018-05-26-serverless-ai-in-my-backyard.md

```diff
@@ -12,11 +12,11 @@ Recently I added some cool stuff to my home automation platform using computer v
 
 ## The 'problem'
 
-When you have kids, you know they leave doors open. In this case, the gate in the fence. This does not have to be a big problem, except when there are bikes unlocked in the backyard, we don't want them to be stolen! 
```

Comment:
Gate in the fence not ok?

Reply:
Gate in the fence = door in the wall. Just discussed.